src/test/java/api/loans/scenarios/ChangeDueDateTests.java

```diff
@@ -1,265 +0,0 @@
-package api.loans.scenarios;
-
-import static api.support.http.InterfaceUrls.loansUrl;
-import static api.support.matchers.PatronNoticeMatcher.hasEmailNoticeProperties;
-import static api.support.matchers.TextDateTimeMatcher.isEquivalentTo;
-import static java.net.HttpURLConnection.HTTP_NO_CONTENT;
-import static org.folio.circulation.support.JsonPropertyWriter.write;
-import static org.hamcrest.CoreMatchers.hasItems;
-import static org.hamcrest.core.Is.is;
-import static org.hamcrest.junit.MatcherAssert.assertThat;
-
-import java.net.MalformedURLException;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.UUID;
-import java.util.concurrent.CompletableFuture;
-import java.util.concurrent.ExecutionException;
-import java.util.concurrent.TimeUnit;
-import java.util.concurrent.TimeoutException;
-
-import org.apache.commons.lang3.StringUtils;
-import org.awaitility.Awaitility;
-import org.folio.circulation.support.http.client.IndividualResource;
-import org.folio.circulation.support.http.client.Response;
-import org.folio.circulation.support.http.client.ResponseHandler;
-import org.hamcrest.Matcher;
-import org.hamcrest.MatcherAssert;
-import org.hamcrest.Matchers;
-import org.joda.time.DateTime;
-import org.joda.time.Period;
-import org.junit.Test;
-
-import api.support.APITests;
-import api.support.builders.ItemBuilder;
-import api.support.builders.LoanPolicyBuilder;
-import api.support.builders.NoticeConfigurationBuilder;
-import api.support.builders.NoticePolicyBuilder;
-import api.support.builders.RequestBuilder;
-import api.support.fixtures.ItemExamples;
-import api.support.fixtures.TemplateContextMatchers;
-import api.support.http.InventoryItemResource;
-import io.vertx.core.json.JsonObject;
-
-public class ChangeDueDateTests extends APITests {
```

Comment:
Why have these tests been removed?

Reply:
Just renamed the class

Reply:
What was this class renamed to? Git / GitHub doesn’t seem to have detected a rename.

Reply:
Possibly `BatchChangeDueDateTests.java`? I see the same methods in there... Seems odd to rename it to this, so maybe I am missing something...

Reply:
@mreno-EBSCO Yeah, I was thinking it was likely that. I was hoping @mrdpopov would confirm that because I prefer not to leap too much. However asynchronous communication often tempts me to do so.

If this was renamed to `BatchChangeDueDateTests` I'd like to understand why. 

As I understand it, the batch operation is a second way of changing the due date, it does not replace the existing way.

If so, I think the tests should be kept separate between the two different APIs.